src/main/java/cam72cam/immersiverailroading/entity/EntityMoveableRollingStock.java

```diff
@@ -344,6 +349,22 @@ public void onTick() {
                         wheel_sound.stop();
                     }
                 }
+                if(MinecraftClient.getPlayer().getPosition().distanceTo(getPosition()) < 40){
+                     if (!flange.isPlaying()) {
+                        flange.play(getPosition());
+                    }
+                    flange.setPitch(pitch + this.sndRand);
+                    flange.setVolume(Math.abs(getFrontYaw()-getRearYaw())/90*adjust);
+
+                    flange.setPosition(getPosition());
+                    flange.setVelocity(getVelocity());
+                    flange.update();                
+                }
+                else {
+                    if (wheel_sound.isPlaying()) {
+                        wheel_sound.stop();
```

Comment:
Was this a copy paste error (should be flange)?  See above comment about breaking wheel_sound.